src/main/java/uk/gov/justice/hmpps/prison/api/resource/PrisonResource.kt

```diff
@@ -43,12 +43,12 @@ class PrisonResource(private val bookingService: BookingService) {
   )
   @Operation(summary = "Details of the active offender booking and calculable sentences at a particular establishment")
   @PreAuthorize("hasRole('RELEASE_DATE_MANUAL_COMPARER')")
-  @GetMapping("/{establishmentId}/booking/latest/sentence-summary")
-  fun getActiveOffenderBookingsByEstablishment(
+  @GetMapping("/{establishmentId}/booking/latest/calculable-sentence-envelope")
```

Comment:
what is a sentence envelope?  If the endpoint is now changed to return that data instead then would be good to amend the swagger to provide the additional information about what it is

Reply:
are there any active consumers of this endpoint?

Reply:
None at present